### ISSUE TITLE
must-gather timeout after 600 seconds

### ISSUE THREAD

**Issue by jilju** [red-hat-storage/ocs-ci#2728]
Observed this issue in all must-gather run in the following execution - https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/11080/

must-gather may take more than 600 seconds to complete. Time-out should occur only if the command is stuck in some state. But  enough time should be provided if must-gather is still running even after 600 seconds.
```
05:43:55 - MainThread - ocs_ci.ocs.utils - ERROR - Timeout 600s for must-gather reached, command exited with error: 
Command '['oc', '--kubeconfig', '/home/jenkins/current-cluster-dir/openshift-cluster-dir/auth/kubeconfig', 
'adm', 'must-gather', '--image=quay.io/rhceph-dev/ocs-must-gather:latest-4.5', 
'--dest-dir=/home/jenkins/current-cluster-dir/logs
/failed_testcase_ocs_logs_1597381981/test_worker_node_restart_during_pvc_expansion_ocs_logs/ocs_must_gather']' 
timed out after 600 seconds
```

**Comment by jilju** [comment]
Closed via #3245

**Comment by jilju** [comment]
Closed via #3245
